app/src/main/java/com/example/nmc/PendingPaymentPay.java

```diff
@@ -0,0 +1,14 @@
+package com.example.nmc;
+
+import androidx.appcompat.app.AppCompatActivity;
+
```

Comment:
Are you creating 2 activities? I see 2 java files for each activity but just 1 XML file.

Reply:
I will commit the other XML file too .
Yes I did create two different activities

Reply:
Sure, are the both activities entirely different or they just differ by one textview? You can just make one activity if that's the case.

Reply:
They are the same. Only with a difference in textview. Okay then just need to need .java file for the other one